int64 robot_id
bool ready
float64[] position
geometry_msgs/Quaternion orientation